spec/index.bs

```diff
@@ -7960,6 +7960,25 @@ dictionary GPURenderPassColorAttachment {
         render pass. If not [=map/exist|provided=] defaults to `{r: 0, g: 0, b: 0, a: 0}`. Ignored
         if {{GPURenderPassColorAttachment/loadOp}} is not {{GPULoadOp/"clear"}}.
 
+        The members of {{GPURenderPassColorAttachment/clearValue}} are all double values, so
+        they will first be converted to the fully qualified format type of
+        {{GPURenderPassColorAttachment/view}} before being set as the clear value of
+        {{GPURenderPassColorAttachment/view}}.
+
+        Let |colorAttachmentFormat| be
+        {{GPURenderPassColorAttachment/view}}.{{GPUTextureView/[[descriptor]]}}.{{GPUTextureViewDescriptor/format}}.
+        - If the components of |colorAttachmentFormat| are floating point values, the members of
+            {{GPURenderPassColorAttachment/clearValue}} will be converted into floating point
+            values.
```

Comment:
The clearValue is already floating point values (64-bit). The conversion is between different precisions of floating point values (e.g. f64->f16 or f64->rg11b10ufloat) and there's still ambiguity about a few things:
- is it truncated or rounded, and if so how
- what happens on underflow (f64 is smaller than the smallest target non-denormal value)
- what happens on overflow (f64 is larger than the largest target finite value)

The WebIDL conversions for f64->f32 for example specify this in detail:
https://webidl.spec.whatwg.org/#es-unrestricted-float

All that to say, we need to write our own conversion rules (similar to WebIDL's but generalized - as mentioned here https://github.com/gpuweb/gpuweb/issues/965#issuecomment-675072177) and we need to verify/test they actually work in the native APIs (results written aren't off-by-one).

Reply:
I think we can always convert the clear values into the ones that can be accepted by all the backends before applying these clear values in the native APIs. For floating point formats, we can always convert the clear values into 32-bit-float ones, which are accepted by all the native APIs (especially on D3D12).

Reply:
For float formats we still have to define how to get all the way from a clear-value to what bits get put in the texture. So for example we need to fully define a conversion to float16 (which webidl doesn't have, but we can use the same rules).